calico_containers/tests/st/test_base.py

```diff
@@ -46,3 +50,28 @@ def start_etcd(self):
             initial_cluster="calico=http://%s:2380" % self.ip,
             initial_cluster_state="new",
         )
+
+    def assert_can_ping(self, workload, ip, retries=5):
```

Comment:
I'd rather have a default of 1 (or maybe 2) as the retry.  In places where the tester knows the network needs time to converge, they should set a higher value.

1 out of 5 packets successful just seems like too low of a default bar to pass and could mask route flapping or other "flaky network" issues.  The underlying network should be rock-solid at this level of utilization, so flakiness is very likely to have been caused by us.